src/workerRunner.ts

```diff
@@ -484,7 +484,10 @@ function buildTestEndPayload(testId: string, testInfo: TestInfo): TestEndPayload
 
 const kTestArgsSymbol = Symbol('testargs');
 
-function modifier(testInfo: TestInfo, type: 'skip' | 'fail' | 'fixme' | 'slow', arg?: boolean | string | Function, description?: string) {
+function modifier(testInfo: TestInfo, type: 'skip' | 'fail' | 'fixme' | 'slow', args: [arg?: boolean | string | Function, description?: string]) {
+  if (args.length >= 1 && !args[0])
```

Comment:
I'd prefer to rewrite the whole section now, like `if (args.length === 0) {} else if (args.length === 1) {} else {}`